### ISSUE TITLE
The full info string for code blocks is not returned

### ISSUE THREAD

**Issue by jonludlam** [ocaml/omd#176]
When parsing [example-113](https://spec.commonmark.org/0.29/#example-113) only the first word of the info string is returned:

```
# Omd.of_string {|
~~~~    ruby startline=3 $%@#$
def foo(x)
  return 3
end
~~~~~~~
|};;
- : Omd.t =
[Omd.Code_block (Some (Omd.Tilde, "ruby"), Some "def foo(x)\n  return 3\nend")]
```

**Comment by nojb** [comment]
Yes, this is according to the spec, where the first word is the "language" of the code block. I guess the rest of the string could be returned to be interpreted by the library user. Is that what you had in mind?

**Comment by jonludlam** [comment]
Exactly - for [mdx](https://github.com/realworldocaml/mdx) for example, where there are number of uses of the rest of the string.
